discovery/consul/consul.go

```diff
@@ -20,6 +21,8 @@ func init() {
 type Consul struct {
```

Comment:
```go
// Consul is a service discovery backend for Hashicorp Consul
type Consul struct
```

Nit but shouldn't we provide a more specific description of what this struct represents within the context of ContainerPilot itself? This is a client wrapper of a tool specific to a set of needs and not necessarily the tool itself.

Also, is the line number wrong on this? ::shrug:: Weird formatting on GH.